lib/bore.lua

```diff
@@ -154,6 +154,7 @@ local function expand(minPosition, maxPosition, shaft)
 end
 
 m.go = function(position, depth, minPosition, maxPosition)
+  wants.setProfile()
```

Comment:
If `setProfile` selects the default by default, then this is unnecessary. I would remove this combined with the other change.

Reply:
agreed